src/containers/shared/components/Transaction/SetRegularKey/parser.ts

```diff
@@ -0,0 +1,5 @@
+export function parser(tx: any) {
+  return {
+    key: tx.RegularKey || null,
```

Comment:
Isn't this already going to be undefined if `tx.RegularKey` doesn't exist?

Reply:
I can try that.  This was copied over from what was there before.